packages/aws-cdk-lib/region-info/lib/default.ts

```diff
@@ -1,12 +1,15 @@
 /**
  * Provides default values for certain regional information points.
+ * @deprecated - Service principals are now globally `<SERVICE>.amazonaws.com`
```

Comment:
What's the direct replacement class / object / functionality that users should use instead of `Default`?

Reply:
I forgot to update this. Will change.